reflex/state.py

```diff
@@ -1218,6 +1220,52 @@ def dict(self, include_computed: bool = True, **kwargs) -> dict[str, Any]:
         variables = {**base_vars, **computed_vars, **substate_vars}
         return {k: variables[k] for k in sorted(variables)}
 
+    @classmethod
+    def _builtin_event_handlers(cls) -> dict[str, Callable]:
```

Comment:
i think this can go away now and we can just define `on_load_internal` directly in `rx.State`, now that it's a subclass of `rx.BaseState`